examples/syscalls.c

```diff
@@ -0,0 +1,93 @@
+
+
+// This file contains examples of system calls performed in standard C.
+
+// To call these, you must include the following header files.
+
+#include <unistd.h>
+#include <sys/syscall.h>
+
+
+
+
+
+
```

Comment:
Number of line breaks between examples in both files seem arbitrary to me. Maybe it's better to clean it up a bit and keep consistent.